File: Programmers/Lv.1/2016년/12901.java
import java.time.DayOfWeek;
import java.time.LocalDate;

class Solution {
    
    final String[] DAY_OF_WEEK = {"", "MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"};
    
    public String solution(int a, int b) {
        String answer = "";
        
        LocalDate date = LocalDate.of(2016, a, b);
        DayOfWeek dayOfWeek = date.getDayOfWeek();
        
        int dayOfWeekNumber = dayOfWeek.getValue();
        answer = DAY_OF_WEEK[dayOfWeekNumber];
        
        return answer;
    }
}

File: Programmers/Lv.1/2016년/12901.js
function solution(a, b) {
    const DAY_OF_WEEK = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
    const answer = DAY_OF_WEEK[new Date(2016, a-1, b).getDay()];    
    return answer;
}
